docs/workflow/building/coreclr/nativeaot.md

```diff
@@ -34,6 +34,14 @@ Run `build[.cmd|.sh] -c Release` from the repo root to build the NativeAOT toolc
 
 * Add the package directory to your `nuget.config` file. For example, add `<add key="local" value="C:\runtime\artifacts\packages\Release\Shipping" />`
 * Run `dotnet add package Microsoft.DotNet.ILCompiler -v 10.0.0-dev` to add the local package reference to your project.
+* Add the following snippet to your project file, replacing `win-x64` with the target runtime identifier:
+
+```
+<ItemGroup>
+   <PackageDownload Include="Microsoft.NETCore.App.Runtime.NativeAOT.win-x64" Version="[10.0.0-dev]" />
+</ItemGroup>
+```
```

Comment:
Similar instructions are also at https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/docs/compiling.md?plain=1#L20-L32 . They need to be updated as well.